packages/core/src/presets.ts

```diff
@@ -27,3 +27,14 @@ export const vercel = createEnv({
   },
   runtimeEnv: process.env,
 });
+
+/**
+ * @see https://docs.uploadthing.com/getting-started/appdir#add-env-variables
+ */
+export const uploadthing = createEnv({
+  server: {
+    UPLOADTHING_SECRET: z.string(),
+    UPLOADTHING_APP_ID: z.string(),
```

Comment:
this variable isn't actually required so can be optional

Reply:
done

Reply:
I meant just appId was optional. The secret is required 😅

Reply:
there u go 👀